components/Article/BlockQuote/index.tsx

```diff
@@ -0,0 +1,8 @@
+import type { PropsWithChildren } from 'react';
+import styles from './index.module.scss';
+
+const BlockQuote = ({ children }: PropsWithChildren) => (
+  <div className={styles.blockQuote}>{children}</div>
```

Comment:
There is a semantic element for this https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

Reply:
Just realized, when using the `blockquote` semantic element, the component styles conflicts with the ones in [styles/_base.scss#L89](https://github.com/nodejs/nodejs.org/blob/major/website-redesign/styles/_base.scss#L89)

Reply:
Feel free to go back to using `div`